agent/src/softwarecontaineragent.h

```diff
@@ -192,7 +192,7 @@ class SoftwareContainerAgent
      *
      * Capabilities are a mapping from a string to a set of gateway configurations. Capabilities
      * are translated into gateway configurations and then sent the same way as with
-     * setGatewayConfigs
+     * startGateways
```

Comment:
Not sure I understand why this sentence mentions the function name, but I just changed it to use the new function name.